pkg/models/dashboards.go

```diff
@@ -224,6 +224,10 @@ type SaveDashboardCommand struct {
 	Result *Dashboard `json:"-"`
 }
 
+type ValidateDashboardCommand struct {
+	Dashboard string `json:"dashboard" binding:"Required"`
```

Comment:
🤔 It would be kinda nice to not have to wrap it in a string, but maybe.
I believe you can skip the string and use the byte slice directly though, which is what you want?

```suggestion
	Dashboard []byte `json:"dashboard" binding:"Required"`
```

Reply:
my rationale behind requiring to send a string was to the API can return line position numbers for an error and it'll map correctly..... though I'm less convinced of this now 😅

Reply:
ahh - it looks like the json parser needs the `string` type to parse it correctly. otherwise i guess it looks for a json array of numbers?